arelle/plugin/validate/NL/rules/br_kvk.py

```diff
@@ -20,6 +20,7 @@
 from ..DisclosureSystems import (
     DISCLOSURE_SYSTEM_NT16,
     DISCLOSURE_SYSTEM_NT17,
+    DISCLOSURE_SYSTEM_NT18
```

Comment:
We also don't have testcases for any of these in our conformance suite

Reply:
Ill write up a ticket for that.